snuba/datasets/entities/factory.py

```diff
@@ -65,7 +71,13 @@ def __initialize(self) -> None:
             }
         )
 
-        # TODO: load the yaml entities here
+        if settings.PREFER_PLUGGABLE_ENTITIES:
+            self._entity_map.update(
+                {
+                    key: build_entity_from_config(path)
+                    for (key, path) in entity_to_config_path_mapping.items()
+                }
+            )
```

Comment:
This will require a redeploy to change back and forth won't it? Isn't the point to be able to toggle it on and off? 

Once this global variable gets initialized, it won't be changed for the runtime of the program (except by some threads). If you want to have this be changeable at runtime, that logic should go into the `_ent_factory` method

Reply:
I don't have any intention of changing back and forth. My thought is just that self-hosted users can't get burned if we only do this in prod (and they may never have use for pluggable entities anyway)